docs/core/diagnostics/dumps.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Dumps - .NET
+description: An introduction to dumps in .NET.
+ms.date: 10/12/2020
+---
+
+# Dumps
+
+A dump is a file that contains snapshot of the process at the time they were created and can be useful for examining the state of your application for debugging. Dumps can be used for post-mortem debugging of your .NET application when it is difficult to attach a debugger to it (for example, when the app is running in production). Using dumps allows you to capture the state of the problematic process and examine it without having to stop it from running while being analyzed.
+
+## Collect dumps
```

Comment:
The TOC generated at the beginning of will use this title. Is something like "Collecting Dumps" or "How to Collect Dumps" more readable? I'm asking mostly from what's the usage in docs vs. what's more readable.

Reply:
This is the correct format. We are not supposed to use -ing words (gerunds).